src/applications/personalization/profile/tests/selectors.unit.spec.js

```diff
@@ -1,5 +1,6 @@
 import { expect } from 'chai';
 import * as selectors from '../selectors';
+import localStorage from 'platform/utilities/storage/localStorage';
```

Comment:
nit: ideally we should prefix these imports with `~/`, so `~/platform`, just so no one thinks that `platform` is a module in node_modules